lib/read_file.js

```diff
@@ -6,7 +6,7 @@ function promisfy (mod, fn) {
     return new Promise(function (resolve, reject) {
       args.push(function (err, res) {
         if (err) reject(err)
-        else resolve(res)
+        else resolve(res.toString())
```

Comment:
@dickeyxxx I refactored to pass `utf-8` in `02459b9`.  It is much better this way, thanks for catching.